sqld/proto/proxy.proto

```diff
@@ -13,6 +13,7 @@ message Query {
         Positional positional = 2;
         Named named = 3;
     }
+    bool skip_rows = 4;
```

Comment:
I can't resist some bikeshedding: why the `skip_rows` and `want_rows` distinction? I generally prefer positive boolean names, like want_rows, because they're more natural to understand. But more importantly, it would be way easier to understand if the approach is unified, and you just use either `want_rows` or `skip_rows` in both rpc and the code, to avoid situations like:
```
skip_rows: !query.want_rows,
```
```
want_rows: !query.skip_rows,
```

If there's a reason for `skip_rows` to exist that I missed, I'm happy to ack, but I'd like to make sure there's a reason first :innocent:

Reply:
k, I'm convinced